src/Jackalope/Node.php

```diff
@@ -1564,12 +1575,24 @@ public function getIterator()
      * @see Node::refresh
      * @see Node::__construct
      */
-    protected function _setProperty($name, $value, $type, $internal)
+    protected function loadProperty($name)
     {
+        if (isset($this->properties[$name])) {
+            return $this->properties[$name];
+        }
+
+        if (!isset($this->propertyValues[$name])) {
+            return;
```

Comment:
is this the expected behaviour? this makes the method return a mix of property and null. could we throw the not existing exception here?